### DISCUSSION TITLE
建议：rerank功能开源社区参与完善

### DISCUSSION THREAD

**Discussion by gaord** [labring/FastGPT#654] [Ideas]
目前的向量搜索有排序问题，导致知识问答出现答非所问的情况。影响客户体验。rerank模型应该可以解决。建议放出来已有实现，社区参与一同完善。

**Comment by qianyvanjian** [comment]
同意，但是现在rerank接入太费劲，有成功的吗？

**Comment by c121914yu** [comment]
> 同意，但是现在rerank接入太费劲，有成功的吗？

应该没有不成功的吧，人家说的时候都没开源，现在镜像都有了。

**Reply by qianyvanjian** [reply]
我也有镜像，但是部署没反应呀，有部署成功的吗？
